dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -1,10 +1,43 @@
 class RubricsController < ApplicationController
+  include Rails.application.routes.url_helpers
+
   def edit
     @rubric = Rubric.find_by(id: params[:id])
     @lesson = @rubric.lesson
   end
 
   def new
     @lesson = Lesson.find_by(id: params[:lessonId])
+    @rubric = Rubric.new
+  end
+
+  def create
+    @rubric = Rubric.new(rubric_params)
+    @lesson = @rubric.lesson
+    if @rubric.save
```

Comment:
oh, one thing I should've mentioned -- you'll need to call `@rubric.lesson.unit.write_script_json` when the save succeeds. This is how we get content from levelbuilder to production (you'll need to do this on update as well when you get there). Happy to chat more about this!